tests/unit/data/aquacomms/aquacomms1.eml

```diff
@@ -1,91 +1,91 @@
-vetica, sans-serif" size=3D"2">Dear Network to Code,<br><br></font></td>
-      </tr>
-      <tr>
-        <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, Hel=
-vetica, sans-serif" size=3D"2">This notice is being sent to notify you of t=
-he following Maintenance Event.<br><br></font></td>
-      </tr>
-
-    </tbody></table>
-    <table border=3D"0" cellpadding=3D"0" cellspacing=3D"0">
-      <tbody><tr>
-        <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, Hel=
-vetica, sans-serif" size=3D"2"><b> Ticket Number: </b></font></td>
-        <td valign=3D"TOP" align=3D"LEFT" style=3D"padding-left:10px"><font=
- face=3D"Arial, Verdana, Helvetica, sans-serif" size=3D"2"> 11111</font></t=
-d>
-      </tr>
-      <tr>
-        <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, Hel=
-vetica, sans-serif" size=3D"2"><b> Scheduled Start Date &amp; Time: </b></f=
-ont></td>
-        <td valign=3D"TOP" align=3D"LEFT" style=3D"padding-left:10px"><font=
- face=3D"Arial, Verdana, Helvetica, sans-serif" size=3D"2"> 22:00 12/10/202=
-0 GMT</font></td>
-      </tr>
-      <tr>
-        <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, Hel=
-vetica, sans-serif" size=3D"2"><b> Scheduled End Date &amp; Time: </b></fon=
-t></td>
-        <td valign=3D"TOP" align=3D"LEFT" style=3D"padding-left:10px"><font=
- face=3D"Arial, Verdana, Helvetica, sans-serif" size=3D"2"> 08:00 13/10/202=
-0 GMT</font></td>
-      </tr>
-     =20
-      <tr>
-        <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, Hel=
-vetica, sans-serif" size=3D"2"><b> Service ID: </b></font></td>
-        <td valign=3D"TOP" align=3D"LEFT" style=3D"padding-left:10px"><font=
- face=3D"Arial, Verdana, Helvetica, sans-serif" size=3D"2">111-AAA-11-11BB-=
-33333</font></td>
-      </tr>
-     =20
-    </tbody></table>
-    <br>
-    <table border=3D"0" cellpadding=3D"0" cellspacing=3D"0">
-      <tbody><tr>
-        <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, Hel=
-vetica, sans-serif" size=3D"2">Please note that all time stamps below will =
-appear in US Central time.<br><br></font></td>
-      </tr>
-     =20
-       =20
-        <tr>
-          <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, H=
-elvetica, sans-serif" size=3D"2">- Update - There are no new updates.<br></=
-font></td>
-        </tr>
-       =20
-     =20
-      <tr>
-          <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, H=
-elvetica, sans-serif" size=3D"2"><br></font></td>
-      </tr>
-      <tr>
-        <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, Hel=
-vetica, sans-serif" size=3D"2">Sincerely,</font></td>
-      </tr>
-      <tr>
-        <td valign=3D"TOP" align=3D"LEFT"><font face=3D"Arial, Verdana, Hel=
-vetica, sans-serif" size=3D"2">Aqua Comms NOC</font></td>
-      </tr>
-    </tbody></table>
-  </font>
-
---000000000000015b7605ca799cf2--
+MIME-Version: 1.0
```

Comment:
Why are the entire files showing as diffs? Did we change newline formats or encoding or something?

Reply:
Looks like this is default behavior of `open()`, which `fileinput.input()` uses under the hood:

> When reading input from the stream, if `newline` is None, universal newlines mode is enabled. Lines in the input can end in '\n', '\r', or '\r\n', and these are translated into '\n' before being returned to the caller.

It's probably desirable in the long run; it just makes it hard to tell from the diffs for this specific PR exactly what's been changed.